framework/web/ErrorAction.php

```diff
@@ -98,6 +98,13 @@ public function init()
      */
     public function run()
     {
+        $response = Yii::$app->getResponse();
+        if ($this->exception instanceof HttpException) {
```

Comment:
Yes. I know that. Check tests. In case of action, error handler isn't involved and it's public API so same could be done in real projects. That's why a safeguard.